feel/feelmodels/modelproperties.cpp

```diff
@@ -154,6 +155,18 @@ ModelProperties::ModelProperties( std::string const& filename, std::string const
     {
         LOG(WARNING) << "Model does not have any postprocess\n";
     }
+    auto out = M_p.get_child_optional("Outputs");
+    if ( out )
+    {
+        LOG(INFO) << "Model with outputs\n";
+        if ( !directoryLibExpr.empty() )
+            M_outputs.setDirectoryLibExpr( directoryLibExpr );
+        M_outputs.setPTree( *out );
+    }
+    else
+    {
+        LOG(WARNING) << "Model does not have any outputs\n";
```

Comment:
Remove this WARNING.  There is no reason to have a warning if the output section is not present.

Reply:
should I remove also the warnings for boundary/initial conditions, materials and postprocess ?

Reply:
yes, you can